scripts/sql/apolloconfigdb.sql

```diff
@@ -34,7 +34,7 @@ DROP TABLE IF EXISTS `App`;
 
 CREATE TABLE `App` (
   `Id` int(10) unsigned NOT NULL AUTO_INCREMENT COMMENT '主键',
-  `AppId` varchar(500) NOT NULL DEFAULT 'default' COMMENT 'AppID',
```

Comment:
Please also add a new folder `v210-v220` to [scripts/delta](https://github.com/apolloconfig/apollo/tree/master/scripts/sql/delta) and add the delta scripts there.